app/controllers/Subscriptions.scala

```diff
@@ -139,7 +142,17 @@ class Subscriptions(
     val css = "paperSubscriptionLandingPage.css"
     val canonicalLink = Some(buildCanonicalPaperSubscriptionLink())
 
-    Ok(views.html.main(title, id, js, css, None, canonicalLink)).withSettingsSurrogateKey
+    def pricesToMap(prices: List[ZuoraCatalogPricePlan], prefix: String) = {
+      prices.map(price => (
+        s"$prefix-${price.name.getOrElse(price.id).toLowerCase}",
```

Comment:
How does the format of the key here relate?

Reply:
The keys are [hardcoded in JS](https://github.com/guardian/support-frontend/pull/1233/files#diff-5d0077fde2361c7fce60450a4338b000R40). It's a bit ugly but the other parts of this project that fetch things from the scala side use `window.{var}` globals which are also hardcoded and a bit more feeble.

A nicer way to do this code wise could be to fetch this from an api-like endpoint but that involves a network round trip. I'm considering passing the whole object as a JSON in the data attr but the keys would still have to be hardcoded in JS to define the shape of the final object